docs-ref-conceptual/authenticate-azure-cli-preview.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Use the subscription selector when logging into Azure.
+description: Learn how to select a subscription from the new subscription selector available with the `az login` command.
+author: dbradish-microsoft
+ms.author: dbradish
+ms.date: 04/09/2024
+ms.topic: conceptual
+ms.service: azure-cli
+ms.tool: azure-cli
+ms.custom: devx-track-azurecli
+keywords: az login, authentication types, authentication methods, azure, cli login, az login powershell, cli login, sign in, azure cli  
+---
+
+# Choose your subscription when logging into Azure using Azure CLI
+
+With the preview release of Azure CLI [2.59.0](./release-notes-azure-cli.md#april-02-2024), there's a new login feature that provides a list of available tenants and subscriptions. Choose to keep the subscription and tenant that is selected by default, or type the line number of the subscription you want to use. Here are the details:
+
+* The subscription selector is available in a preview version of Azure CLI 64-bit Windows, Linux, or macOS.
+* The subscription selector is only available when using the `az login` command [interactively](./authenticate-azure-cli-interactively.md).
+* The authentication preview must be installed to use the subscription selector.
+* The preview is installed in the directory you specify, and coexists with your current Azure CLI installation.
+* The authentication preview doesn't prompt you to select a subscription when you're logging in with a service principal, managed identity, or Web Account Manager (WAM).
+
+With the current practice, the [az account set](/cli/azure/account#az-account-set) command is required to change your subscription. With the subscription selector at time of interactive login, you skip having to [list](/cli/azure/account#az-account-list), [set](/cli/azure/account#az-account-set) and [show](/cli/azure/account#az-account-show) your default subscription.
+
+## Prerequisite for macOS
+
+If you're installing the preview in a macOS environment, Python runtime must be installed.
+
+```bash
+mkdir azure-cli-edge-build 
+cd azure-cli-edge-build 
+git clone https://github.com/Azure/azure-cli --branch login --depth 1 
+python3 -m venv env 
+. env/bin/activate 
+pip install azdev 
+azdev setup -c 
+az login 
+```
+
+## Install
```

Comment:
Those installation steps are for Windows, they appear to also be for MacOs in the current layout of the page.